Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml

```
@@ -2896,6 +2896,672 @@ script:
+      description: Indicate who last modified the rule.
+      type: String
+    - contextPath: CrowdStrike.MLExclusion.created_on
+      description: Indicate who created the rule.
+      type: Date
+    - contextPath: CrowdStrike.MLExclusion.created_by
+      description: The date when the exclusion rule was created.
+      type: String
+  - arguments:
+      - description: Delete the ML exclusions by id
+        defaultValue: The ID of the exclusion to update.
+        name: ids
+        required: true
+        isArray: true
+    description: Delete the ML exclusions by id.
+    name: cs-falcon-delete-ml-exclusion
+  - arguments:
+    - description: A custom filter in which the exclusions should be filtered. For example `value:”<value>”`.
+      name: filter
+      required: false
+    - description: The value in which the exclusions should be filtered.
+      name: value
+      required: false
+    - description: The IDs of the exclusions to retrieve. IDs overwrites the filter and value.
+      name: ids
+      required: false
+      isArray: true
+    - description: The maximum number of records to return. [1-500]
+      name: limit
+      required: false
+    - description: The offset to start retrieving records from.
+      name: offset
+      required: false
+    - auto: PREDEFINED
+      description: Hwo to Sort the retrieved exclusions.
+      name: sort
+      required: false
+      predefined:
+      - applied_globally.asc
+      - applied_globally.desc
+      - created_by.asc
+      - created_by.desc
+      - created_on.asc
+      - created_on.desc
+      - last_modified.asc
+      - last_modified.desc
+      - modified_by.asc
+      - modified_by.desc
+      - value.asc
+      - value.desc
+    description: Get a list of ML Exclusions by specifying their IDs, value, or a specific filter.
+    name: cs-falcon-search-ml-exclusion
+    outputs:
+    - contextPath: CrowdStrike.MLExclusion.id
+      description: The ML exclusion ID.
+      type: String
+    - contextPath: CrowdStrike.MLExclusion.value
+      description: The ML exclusion value.
+      type: String
+    - contextPath: CrowdStrike.MLExclusion.regexp_value
+      description: A regular expression for matching the excluded value.
+      type: String
+    - contextPath: CrowdStrike.MLExclusion.value_hash
+      description: An hash of the value field.
+      type: String
+    - contextPath: CrowdStrike.MLExclusion.excluded_from
+      description: What the exclusion applies to (e.g. a specific ML model).
+      type: String
+    - contextPath: CrowdStrike.MLExclusion.groups.id
+      description: Groups ID that the exclusion rule is associated with.
+      type: String
+    - contextPath: CrowdStrike.MLExclusion.groups.group_type
+      description: Groups type that the exclusion rule is associated with.
+      type: String
+    - contextPath: CrowdStrike.MLExclusion.groups.name
+      description: Groups name that the exclusion rule is associated with.
```

same as previous comment
```suggestion
      description: Groups name that the exclusion rule is associated with.
```